### PR TITLE
Feature/css handles

### PR DESCRIPTION
**Pull request by mariusfiliutamindcell**
In order to root elements to be stylized a css handles on the default approach of vtex had to be applied

### CODE REVIEW

**Review by klzns — APPROVED** [review]

**Comment by klzns** [comment]
Tested in Als
https://brenovtex--alssports.myvtex.com/nike-shoe-air-zoom-pegasus-36-mens/p

**Review by kaisermann — APPROVED** [review]

**Comment by klzns** [comment]
Thanks @mariusfiliutamindcell! We will publish this on Tuesday.

File: react/RatingSummary.tsx [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
+  appSettings: Settings
 }
-const withSettings = graphql<{ client: any }, Settings>(getConfig, {
-  options: () => ({ ssr: false }),
-})
 
-const RatingSummary: FunctionComponent<
-  ChildProps<Partial<RatingSummaryProps>, Settings>
-> = props => {
+const RatingSummary: FC<Props> = ({ appSettings }) => {
```
Comment (mariusfiliutamindcell) [inline react/RatingSummary.tsx:14]: Does this work out of the box, without injecting the appSettings ( somehow ) in the Component's props? I tried this approach in another project and it seems to be undefined.

Comment (klzns) [reply react/RatingSummary.tsx:14]: Have you tried adding this to the app settings property:

image

This is injected automatically to all the blocks of this app. Maybe you need to use the function `getSettings` from `vtex.render-runtime`

```
import { useRuntime } from 'vtex.render-runtime'

const Foo = () => {
  const { getSettings } = useRuntime()
  const settings = getSettings('yourapp.name')
```

Comment (mariusfiliutamindcell) [reply react/RatingSummary.tsx:14]: I had set access to public and tried useRuntime but "getSettings" is undefined. Any other advices?

**Pull request merged**